keyboards/crkbd/keymaps/kidbrazil/keymap.c

```diff
@@ -0,0 +1,227 @@
+#include QMK_KEYBOARD_H
+
+// [Init Variables] ----------------------------------------------------------//
+extern uint8_t is_master;
+// Oled timer similar to Drashna's
+static uint32_t oled_timer = 0;
+// Boolean to store
+bool eeprom_oled_enabled = false;
+
+// [CRKBD layers Init] -------------------------------------------------------//
+enum crkbd_layers {
+    _QWERTY,
+    _NUM,
+    _SYM,
+    _GAME,
+    _WEAPON
+};
+
+// [Keymaps] -----------------------------------------------------------------//
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	  [_QWERTY] = LAYOUT(\
+      KC_ESC, KC_Q, KC_W, KC_E, KC_R, KC_T,                                  KC_Y, KC_U, KC_I, KC_O, KC_P, KC_BSPC,\
+      LSFT_T(KC_TAB), KC_A, KC_S, KC_D, KC_F, KC_G,                          KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT,\
+      KC_LCTL, KC_Z, KC_X, KC_C, KC_V, KC_B,                                 KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT,\
+                          LGUI_T(KC_PGUP), MO(_NUM), KC_SPC,         KC_ENT, MO(_SYM), LALT_T(KC_PGDN)\
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.